services/frontend-service/src/ui/components/ReleaseDialog/ReleaseDialog.scss

```diff
@@ -1,29 +1,47 @@
 @use '@material/typography/mdc-typography';
 @use '@material/card/mdc-card';
 
```

Comment:
could you append a screenshot of what it looks like now? It's really hard to review css changes

Reply:
![image](https://user-images.githubusercontent.com/78358035/206173954-cec282d9-adf9-4bfd-b741-3f9d73ea8e8f.png)

Reply:
1) please format the release date as something more common like this: `2014-08-20 15:30`. I don't think we need seconds at all.

2) the sha-id needs a bigger margin (in all 4 directions), please make it look like in the design

Reply:
Is this the same, dark blue that we are using?

Reply:
Fixed it now 
![image](https://user-images.githubusercontent.com/78358035/206181432-97cb3714-7904-43f9-869d-0a8ae3bb0776.png)